src/rhsmlib/dbus/objects/register.py

```diff
@@ -275,12 +377,18 @@ def RegisterWithActivationKeys(self, org, activation_keys, options, connection_o
             Locale.set(locale)
             cp = self.build_uep(connection_options)
 
+            # Remove 'enable_content' option, because it will not be proceed in register service
```

Comment:
I'm not sure I understand. The 'enable_content' option won't be used in the registration service?

Reply:
This option will not be used in registration service, because current REST API endpoint does not know this option.

Reply:
Maybe we could do some refactoring of move the same logic from sub-man CLI and D-Bus methods to the service, but I didn't want to make this PR more complicated than necessary.